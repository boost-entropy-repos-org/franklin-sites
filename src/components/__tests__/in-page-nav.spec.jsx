import React from 'react';
import renderWithRouter from '../../testHelpers/renderWithRouter';
import InPageNav from '../in-page-nav';

describe('InPageNav component', () => {
  test('should render', () => {
    const sections = [
      {
        id: 'id1',
        label: 'First link',
      },
      {
        id: 'id2',
        label: 'Second link',
      },
      {
        id: 'id3',
        label: 'Third link',
        disabled: true,
      },
    ];

    const { asFragment } = renderWithRouter(<InPageNav sections={sections} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
