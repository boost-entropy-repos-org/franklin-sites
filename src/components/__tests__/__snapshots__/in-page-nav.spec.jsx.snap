// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InPageNav component should render 1`] = `
<DocumentFragment>
  <div
    class="in-page-nav"
  >
    <ul
      class=""
    >
      <li
        class=""
      >
        <a
          href="/#id1"
        >
          First link
        </a>
      </li>
      <li
        class=""
      >
        <a
          href="/#id2"
        >
          Second link
        </a>
      </li>
      <li
        class="in-page-nav--disabled"
      >
        <a
          href="/#id3"
        >
          Third link
        </a>
      </li>
    </ul>
  </div>
</DocumentFragment>
`;
